Gifski/EditVideoViewController.swift

```diff
@@ -299,6 +299,19 @@ final class EditVideoViewController: NSViewController {
 		}
 	}
 
+	private func showConversionCompletedAnimationWarningIfNeeded() {
+		// TODO: This function can be removed once NSImageView respects the looping counter in the GIF header.
+		SSApp.runOnce(identifier: "convCompleteWarning") {
+			DispatchQueue.main.async { [self] in
+				NSAlert.showModal(
+					for: view.window,
+					message: "Animated GIF Preview Limitation",
+					informativeText: "The Conversion Completed Preview after processing will not reflect the image repetitions selected. However, the exported file will refelct the options selected and repeat correctly."
```

Comment:
I think this could be more succinctly written.

And as I commented earlier, Quick Look ignores the loop count and just always loops forever, which should also be mentioned.

"image repetitions" => "loop count"

Also typo.